src/tsickle.ts

```diff
@@ -1119,7 +1119,8 @@ class Annotator extends ClosureRewriter {
               (exp.sym.flags & ts.SymbolFlags.Value) === 0;
       if (!isTypeAlias) continue;
       const typeName = this.symbolsToAliasedNames.get(exp.sym) || exp.sym.name;
-      this.emit(`/** @typedef {${typeName}} */\nexports.${exp.name}; // re-export typedef\n`);
+      // Leading newline prevents the typedef from being swallowed.
```

Comment:
Load bearing leading whitespace. Sigh. But this is the right fix, until we port all of this to transformers.